asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py

```diff
@@ -293,115 +452,280 @@ def _generate_poses(
 
         return posed_ligands, failed_ligands
 
-    def _prune_clashes(self, receptor: oechem.OEMol, ligands: list[oechem.OEMol]):
+
+class RDKitConstrainedPoseGenerator(_BasicConstrainedPoseGenerator):
+    """Use RDKit to embed multiple conformers of the molecule while constraining it to the template ligand."""
+
+    type: Literal["RDKitConstrainedPoseGenerator"] = "RDKitConstrainedPoseGenerator"
+
+    max_confs: PositiveInt = Field(
+        300, description="The maximum number of conformers to try and generate."
```

Comment:
is this something we could reduce by default to speed things up? where did you find 300?

Reply:
Yeah I have been looking into this and 300 comes from this paper looking into the rdkit conformer generator https://www.rdkit.org/UGM/2012/Ebejer_20110926_RDKit_1stUGM.pdf.

Reply:
ok, good to stick with that for now!